MVC-App/Domain.Models/RelationViewModel.cs

```diff
@@ -32,10 +37,14 @@ public class RelationViewModel
 
```

Comment:
For string proprs set StringLength attribure with maximumLength: 128 for example.